metricbeat/module/coredns/_meta/kibana/7/dashboard/Metricbeat-coredns-overview.json

```diff
@@ -0,0 +1,1395 @@
+                },
+                "name": "LeftAxis-1",
+                "position": "left",
+                "scale": {
+                  "mode": "normal",
+                  "type": "linear"
+                },
+                "show": true,
+                "style": {},
+                "title": {
+                  "text": "Size Bytes"
+                },
+                "type": "value"
+              }
+            ]
+          },
+          "title": "Request Size Bytes [Metricbeat CoreDNS] ECS",
+          "type": "line"
+        }
+      },
+      "id": "45d03ed0-53f1-11e9-b466-9be470bbd327-ecs",
+      "type": "visualization",
+      "updated_at": "2019-04-25T22:47:01.782Z",
+      "version": "WzI3LDFd"
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "index": "metricbeat-*",
+            "query": {
+              "language": "lucene",
+              "query": ""
+            }
+          }
+        },
+        "title": "Requests by Zone [Metricbeat CoreDNS] ECS",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [
+            {
+              "enabled": true,
+              "id": "1",
+              "params": {},
+              "schema": "metric",
+              "type": "count"
+            },
+            {
+              "enabled": true,
+              "id": "2",
+              "params": {
+                "field": "coredns.stats.zone",
+                "missingBucket": false,
+                "missingBucketLabel": "Missing",
+                "order": "desc",
+                "orderBy": "1",
+                "otherBucket": false,
+                "otherBucketLabel": "Other",
+                "size": 5
+              },
+              "schema": "segment",
+              "type": "terms"
+            }
+          ],
+          "params": {
+            "addLegend": true,
+            "addTooltip": true,
+            "isDonut": false,
+            "labels": {
+              "last_level": true,
+              "show": true,
+              "truncate": 100,
+              "values": true
+            },
+            "legendPosition": "top",
+            "type": "pie"
+          },
+          "title": "Requests by Zone [Metricbeat CoreDNS] ECS",
+          "type": "pie"
+        }
+      },
+      "id": "84a59b70-5593-11e9-b87d-f90ad5e8a010-ecs",
+      "type": "visualization",
+      "updated_at": "2019-04-25T22:49:59.564Z",
+      "version": "WzIwOSwxXQ=="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "index": "metricbeat-*",
+            "query": {
+              "language": "lucene",
+              "query": ""
+            }
+          }
+        },
+        "title": " Cache Hits, Misses [Metricbeat CoreDNS] ECS",
```

Comment:
the metric being shown is an always increasing count. What about using Max instead of Avg?